src/seeds.gratitude.cpp

```diff
@@ -70,61 +99,173 @@ ACTION gratitude::acknowledge (name from, name to, string memo) {
         item.receivers.push_back(to);
     });
   }
+
+  // Updates ack stats
+  auto stitr = stats2.rbegin();
+  auto round_id = stitr->round_id;
+
+  auto stitr2 = stats2.find(round_id);
```

Comment:
you don't need sitr2 here at all, stitr == stitr2 ...

1 you are getting the round_id (the primary key) from stitr
2 - you're searching for round_id to get stitr2 -> stitr2 will always be == stitr

Reply:
Oh, sure!